srcpkgs/got/template

```diff
@@ -1,21 +1,17 @@
 # Template file for 'got'
 pkgname=got
-version=0.70
+version=0.71
 revision=1
 wrksrc=got-portable-${version}
 build_style=gnu-configure
-hostmakedepends="automake byacc pkg-config"
+hostmakedepends="byacc pkg-config"
 makedepends="libmd-devel zlib-devel libuuid-devel libbsd-devel ncurses-devel openssl-devel"
 short_desc="VCS which prioritizes ease of use and simplicity over flexibility"
 maintainer="Omar Polo <op@omarpolo.com>"
 license="ISC"
 homepage="https://gameoftrees.org"
```

Comment:
```suggestion
homepage="https://gameoftrees.org"
changelog="http://gameoftrees.org/releases/CHANGES"
```

Reply:
please add that line, but use https if possible

Reply:
Added.  Thanks, I didn't know about this variable! :)